conans/client/cmd/export_linter.py

```diff
@@ -69,6 +69,8 @@ def _accept_message(msg):
             return False
         if symbol == "no-name-in-module" and "python_requires" in msg.get("message"):
             return False
+        if symbol == "no-member" and "conan_data" in msg.get("message"):
```

Comment:
Have you tried adding it in ``transform_conanfile()`` function of the pylint_plugin logic instead?

Reply:
no, but it's a good idea.

Reply:
yeah, much better now! thanks!